### PR TITLE
Dashboard of what data is present in the DB

### PR DESCRIPTION
**Pull request by laurentqro**
screen shot 2018-02-07 at 2 50 55 pm

https://trello.com/c/9T5RNLeN/142-admin-reports-on-find

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -4,17 +4,24 @@ def index
     home_path_for_user
   end
 
-  # FIX: Temporary controller, remove me when no longer required
```
Comment (laurentqro) [inline app/controllers/home_controller.rb]: Temporary ™️

Comment (TobyRet) [reply app/controllers/home_controller.rb]: ha!

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -4,17 +4,26 @@ def index
...
   def quality
     @scores = QualityScore.all.order(median: :desc, highest: :desc)
   end
 
+  def dashboard
```
Comment (TobyRet) [inline app/controllers/home_controller.rb]: Any particular reason why the dashboard is being managed by the HomeController?

File: app/controllers/home_controller.rb [hunk 3]
```diff
@@ -4,17 +4,26 @@ def index
...
+    @datasets_count = Dataset.count
+    @datafiles_count = Link.count
+    @publishers_count = Organisation.count
+    @published_datasets_count = Dataset.published.count
+    @draft_datasets_count = Dataset.draft.count
+    @unpublished_datasets_count = Dataset.left_outer_joins(:datafiles).where(links: { id: nil } ).count # i.e. datasets with no datafiles
```
Comment (TobyRet) [inline app/controllers/home_controller.rb]: Perhaps move to a class method called 'datasets_with_no_datafiles' ?

Comment (laurentqro) [reply app/controllers/home_controller.rb]: good idea

File: app/controllers/home_controller.rb (hunk 2 above)
Comment (laurentqro) [reply app/controllers/home_controller.rb]: I agree, it's weird. This is ripe for a refactor.

I'd rather have a `Pages` controller for all our static pages, like we have in FIND, but this is what we have at the moment.

I thought about refactoring this too, but found it is out of the scope of this PR.

**Pull request merged**